packages/kuma-gui/src/app/x/components/x-badge/XBadge.vue

```diff
@@ -14,3 +14,10 @@ const props = withDefaults(defineProps<{
   maxWidth: 'auto',
 })
 </script>
+
+<style lang="scss" scoped>
+:deep(a) {
+  text-decoration: underline;
+  color: currentColor;
+}
+</style>
```

Comment:
Is this the reason why we have underlines here now:

![Screenshot 2024-12-17 at 12 40 38](https://github.com/user-attachments/assets/73c68d36-3288-4591-9eaf-50f6df8f6dea)

Is that intentional?

Reply:
Yes, I thought it would help the user to identify that there is a link behind. That was a quick decision and I know that it affects all the places. Lmk if this is completely off design 🙂

Reply:
Gotcha, this is definitely something I wouldn't add to this PR then, feels like it changes things all over the place instead of just in the places we are working on here. Umm, is this something you definitely need in this PR?

Reply:
Nope, we can discuss this separately 👍 Another option could be to add an icon to indicate the link. It's just that without any indication, as a user, I don't know that there is a link behind.

Reply:
Removed 🙂